src/cts/test/array.tcl

```diff
@@ -130,6 +130,8 @@ set_wire_rc -signal -layer $wire_rc_layer
 set_wire_rc -clock  -layer $wire_rc_layer_clk
 set_dont_use $dont_use
 
+set_debug_level CTS legalizer 1
```

Comment:
Do you want the debug prints in this unit test? If so, you should probably update the ok file with the debug prints.

Reply:
Sorry, it seems it matters when I do "git add".   The issues should be fixed now.